pkg/runner/terraform/runner.go

```diff
@@ -170,22 +171,25 @@ func (r *terraformRunner) injectStateTypeInstance() error {
 		return runner.SaveToFile(varsFilepath, []byte("\n"))
 	}
 
+	r.log.Debug("Reading TypeInstance with TF state and vars", zap.String("path", r.cfg.StateTypeInstanceFilepath))
 	data, err := ioutil.ReadFile(r.cfg.StateTypeInstanceFilepath)
 	if err != nil {
 		return errors.Wrapf(err, "while reading state file %s", r.cfg.StateTypeInstanceFilepath)
 	}
 
-	state := &StateTypeInstance{}
-	if err := yaml.Unmarshal(data, state); err != nil {
+	state := struct {
+		Value StateTypeInstance `json:"value"`
```

Comment:
I only tested that with the instruction described in README.md. I didn't test that with real Capact manifests.